lib/eslint-tester.js

```diff
@@ -181,7 +188,16 @@ ESLintTester.prototype = {
         });
 
         return result.suite;
+    },
+
+    /**
+     * Gets the name of the last used parser.
+     * @returns {string} The name of the last used parser.
+     */
+    getParser: function() {
```

Comment:
Yes, so we could only know which parser was used last.

Also, I'm getting the information from the configuration and not from ESLint so the test is not really great, but ESLint does not expose any information about the used parser.